scripts/build/termux_step_massage.sh

```diff
@@ -82,7 +82,8 @@ termux_step_massage() {
 			if head -c 100 "$file" | head -n 1 | grep -E "^#!.*/bin/.*" | grep -q -E -v -e "^#! ?/system" -e "^#! ?$TERMUX_PREFIX_CLASSICAL"; then
 				sed --follow-symlinks -i -E "1 s@^#\!(.*)/bin/(.*)@#\!$TERMUX_PREFIX/bin/\2@" "$file"
 			fi
-		done < <(find -L . -type f -print0)
+		# recursively search for all names of text-only files containing `#` symbol at the start of the first line, print NULL-terminated list.
+		done < <(grep -rIlZm1 '^#' .)
```

Comment:
That's some fairly uncommon options that should probably have some additional explanation.
```suggestion
		# get all text files text files containing `#` symbol at the start of the first line
		# -r - recursive
		# -I - ignore binaries
		# -l - output filenames for files with matches
		# -Z - Use a null (\0) terminator to delimit matches
		# -m1 - limit search to 1 match per file
		done < <(grep -r -I -l -Z -m1 '^#' .)
```

Alternatively, consider using the long names for the options.
```suggestion
		done < <(grep --recursive --binary-files=without-match --files-with-matches --null --max-count=1 '^#' .)
```
Or both.